NagorniyAleksandr/SQLTask/src/main/resources/jdbc.properties

```diff
@@ -0,0 +1,6 @@
+# MySQL Properties
```

Comment:
copy jdbc.properties to jdbc.properties and put it to test/resources

define h2 in-memory database for testing